lib/instrumentation/index.js

```diff
@@ -110,6 +110,7 @@ Instrumentation.prototype.addEndedTransaction = function (transaction) {
     var payload = agent._transactionFilters.process(transaction._encode())
     if (!payload) return agent.logger.debug('transaction ignored by filter %o', { trans: transaction.id, trace: transaction.traceId })
     agent.logger.debug('sending transaction %o', { trans: transaction.id, trace: transaction.traceId })
+    agent._metrics.getOrCreateCounter('agent.transactions.sent').inc()
```

Comment:
We decided to delay the introduction of `agent.*` metrics, so this line should just be removed